_storage/overview.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# A decentralized storage architecture
+
+The Blockstack Network stores application data using a storage service called
+Gaia. Transactional metadata is stored on the Blockstack blockchain and user
+application data is stored in Gaia hub. Storing data off of the blockchain
+ensures that Blockstack applications can provide users with high-performance and
+high availability for data reads and writes without introducing central trust
+parties.
+
+* TOC
+{:toc}
+
+##  Understand Gaia in the Blockstack architecture
+
+The following diagram depicts the Blockstack architecture and Gaia's place in it:
+
+{% include architecture.md %}
+
+## User control or how is Gaia decentralized?
+
+Within the provider's storage system a Gaia hub runs as a service. This service
+authenticates writes to a location by requiring a valid authentication token from a requestor.
+
+![Gai storage](/storage/images/gaia-storage.png)
+
+Gaia's approach to decentralization focuses on user control of data and storage.
+If a user can choose which storage provider to store data with, then that is all the decentralization required to enable user-controlled
+applications.
+
+The control of user data lies in the way that user data is accessed.
+When an application fetches a file `data.txt` for a given user `alice.id`, the
+lookup will follow these steps:
+
+1. Fetch the `zonefile` for `alice.id`
+2. Read her profile URL from her `zonefile`.
+3. Fetch Alice's profile.
```

Comment:
Should we add "...as served by a Gaia hub at the profile URL?" or something of that sort?

Reply:
At step 3, it doesn't know where Gaia lives. I don't believe the profile is on the Gaia hub.  The profile is defined with the identity.  Both can be gotten by querying the blockchain explorer. 

@kantai is the profile actually in Gaia storage?

Reply:
Yes, the profile is usually stored in Gaia storage. But you're correct that it is somewhat of a special Gaia file. The blockchain explorer contains the exact URL of the profile file, whereas other Gaia files' URLs are obtained through reading information in the user's profile.

Because of this, if a user had a great command of the CLI, they _could_ store their profile file somewhere else, and still have it point to Gaia storage for their application storage, etc.

Reply:
@kantai Thank you.  At this point, I don't think this detail is necessary here. I do think it might be useful in the profile information. I've made a ticket to address this in the core documentation:

https://github.com/blockstack/docs.blockstack/issues/137